apps/admin/app/students/layout.tsx

```diff
@@ -6,13 +6,18 @@ import { StudyProvider } from "./_contexts/StudyProvider";
 
 const StudentsLayout = ({
   children,
+  modal,
 }: Readonly<{
   children: React.ReactNode;
+  modal: React.ReactNode;
 }>) => {
   return (
     <StudyProvider>
       <Navbar />
-      <styled.main className={StudentsLayoutStyle}>{children}</styled.main>
+      <styled.main className={StudentsLayoutStyle}>
```

Comment:
p4;
컨벤션 통일을 위해 studentsLayoutStyle 로 네이밍 해도 좋을 거 같아요!
className 으로 스타일링 하는 부분이라 그냥 main 태그를 써도 좋을 거 같습니당
```suggestion
      <styled.main className={studentsLayoutStyle}>
```